prep-docker-compose.sh

```
@@ -0,0 +1,83 @@
+      exit 1
+    fi
+    echo "==> Building Docker image with debug profile..."
+    docker build --build-arg BUILD_PROFILE=debug -t "${DOCKER_IMAGE}" .
+  else
+    echo "==> Skipping Docker image build (--skip-build)"
+  fi
+fi
+
+# Generate docker-compose.yaml from template
+echo "==> Generating docker-compose.yaml"
+envsubst '$DOCKER_IMAGE $DATA_VOLUME_PATH $PULL_POLICY $GRAFANA_ADMIN_PASSWORD $SCHWAB_BROKER $ALPACA_BROKER' < docker-compose.template.yaml > docker-compose.yaml
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)